server/src/test/java/io/druid/server/coordinator/rules/LoadRuleTest.java

```diff
@@ -51,13 +57,29 @@
  */
 public class LoadRuleTest
 {
+  private static final Logger log = new Logger(LoadRuleTest.class);
+  private static final ObjectMapper jsonMapper = new DefaultObjectMapper();
+
+  private static final ServiceEmitter emitter = new ServiceEmitter(
+      "service",
+      "host",
+      new LoggingEmitter(
```

Comment:
Do you need to have a logging emitter here? It seems to me it would make more sense to have either something you can capture events, or something that's a NoOp